homeassistant/config_entries.py

```diff
@@ -1360,6 +1360,28 @@ async def async_finish_flow(
                 ir.async_delete_issue(self.hass, HOMEASSISTANT_DOMAIN, issue_id)
 
         if result["type"] != data_entry_flow.FlowResultType.CREATE_ENTRY:
```

Comment:
What other result types do we expect here?

Reply:
`async_finish_flow` is called on `FlowResultType.ABOIRT` and `FlowResultType.CREATE_ENTRY`.
I agree it's not clear, a separate PR could add that to the docstring of `async_finish_flow`, in `FlowManager` and in derived classes?

Reply:
Done in PR https://github.com/home-assistant/core/pull/126178